extensions/amp-consent/0.1/test/test-consent-ui.js

```diff
@@ -337,13 +338,7 @@ describes.realWin(
           ).to.be.false;
 
           // Send expand
-          consentUI.handleIframeMessages_({
-            source: consentUI.ui_.contentWindow,
-            data: {
-              type: 'consent-ui',
-              action: 'enter-fullscreen',
-            },
-          });
+          sendIframeMessage(consentUI, 'enter-fullscreen');
```

Comment:
Can we please not name the function `sendIframeMessage`, it's very confusing with the `this.sendIframeMessage_()` method which sends message from parent to the iframe.